src/ray/raylet/local_object_manager.cc

```diff
@@ -91,7 +101,9 @@ void LocalObjectManager::FlushFreeObjectsIfNeeded(int64_t now_ms) {
 
 int64_t LocalObjectManager::SpillObjectsOfSize(int64_t num_bytes_to_spill,
                                                int64_t min_bytes_to_spill) {
+  RAY_CHECK(object_pinning_enabled_) << "Not Implemented";
```

Comment:
Why do we check here? What will happen if it's not enabled?

I would consider removing the check since it's adding noise to the code.

Reply:
Will remove it. I added it kind of for the testing purpose (just to make sure these code path is not invoked when this flag is on in various tests).